os/Kconfig

```diff
@@ -280,7 +280,7 @@ config BOARD_BUILD_DATE
 		So it should be set to a higer value than versions of old binaries if you want to make the binary the latest version.
 
 config BINARY_SIGNING
-	bool "Enable Binary Signing"
+	bool
```

Comment:
Let's add explanation for this in the help.
"Because binary signing is not supported here, this config is set as n and invisible".